src/appleseed/renderer/kernel/lighting/lighttree.cpp

```diff
@@ -65,20 +65,32 @@ bool LightTree::is_built() const
     return m_built;
 }
 
-void LightTree::build(
-    const std::vector<NonPhysicalLightInfo>&    non_physical_lights)
+size_t LightTree::build(
+        std::vector<NonPhysicalLightInfo>&    non_physical_lights,
```

Comment:
EMTs can't be const because I update them with the LT node index. problem is that then the LightSource within the tree can't be const. And if that is the case, NPLs can't be const as well.